<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VFDTools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="setting_tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>时间设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>时间设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_show_time">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TIME_STR</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_show_ts">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TIME_STAMP</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_sync_time">
             <property name="text">
              <string>同步时间</string>
             </property>
             <property name="icon">
              <iconset resource="pics.qrc">
               <normaloff>:/pics/set.png</normaloff>:/pics/set.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>日期格式设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="2">
              <widget class="QPushButton" name="btn_set_dfmt">
               <property name="text">
                <string>设置</string>
               </property>
               <property name="icon">
                <iconset resource="pics.qrc">
                 <normaloff>:/pics/set.png</normaloff>:/pics/set.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="rb_datefmt1">
               <property name="text">
                <string>格式1</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;格式1 - %y/%m/%d（年/月/日） - 例如 23/11/16&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="rb_datefmt2">
               <property name="text">
                <string>格式2</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;格式2 - %m/%d %u（月/日  星期） - 例如 11/16 4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>设备设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>开机字符串设置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="le_str_edit">
             <property name="text">
              <string>VFDClock</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>(最长8字符)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_set_str">
             <property name="text">
              <string>设置</string>
             </property>
             <property name="icon">
              <iconset resource="pics.qrc">
               <normaloff>:/pics/set.png</normaloff>:/pics/set.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>245</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>屏幕保护设置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>开启：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_beg_hour">
             <property name="suffix">
              <string> (H)</string>
             </property>
             <property name="maximum">
              <number>24</number>
             </property>
             <property name="value">
              <number>6</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string> 关闭：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_end_hour">
             <property name="suffix">
              <string> (H)</string>
             </property>
             <property name="maximum">
              <number>24</number>
             </property>
             <property name="value">
              <number>22</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_set_ssd">
             <property name="text">
              <string>设置</string>
             </property>
             <property name="icon">
              <iconset resource="pics.qrc">
               <normaloff>:/pics/set.png</normaloff>:/pics/set.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>LED设置</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBox" name="cb_led1">
               <property name="text">
                <string>LED1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_led2">
               <property name="text">
                <string>LED2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_led3">
               <property name="text">
                <string>LED3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_set_led">
               <property name="text">
                <string>设置</string>
               </property>
               <property name="icon">
                <iconset resource="pics.qrc">
                 <normaloff>:/pics/set.png</normaloff>:/pics/set.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>亮度设置</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lb_hs_val">
               <property name="text">
                <string>当前值</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="hs_bright">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_bright_set">
               <property name="text">
                <string>设置</string>
               </property>
               <property name="icon">
                <iconset resource="pics.qrc">
                 <normaloff>:/pics/set.png</normaloff>:/pics/set.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>日志监控</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="tb_log">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Hello VFDTools Qt  V1.0 - Powered By AQin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_port_list"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_open_port">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_rescan">
        <property name="text">
         <string>扫描串口</string>
        </property>
        <property name="icon">
         <iconset resource="pics.qrc">
          <normaloff>:/pics/scan.png</normaloff>:/pics/scan.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_clear_log">
        <property name="text">
         <string>清空日志</string>
        </property>
        <property name="icon">
         <iconset resource="pics.qrc">
          <normaloff>:/pics/clear.png</normaloff>:/pics/clear.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/exit.png</normaloff>:/pics/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear_log</sender>
   <signal>clicked()</signal>
   <receiver>tb_log</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
